common/common_test.go

```diff
@@ -856,3 +856,23 @@ func TestErrorCollector(t *testing.T) {
 	require.True(t, ok, "Must return a multiError")
 	assert.Len(t, errs.Unwrap(), 2, "Should have 2 errors")
 }
+
+// TestBatch exercises the Batch function
+func TestBatch(t *testing.T) {
+	s := []int{1, 2, 3, 4, 5, 6, 7}
+	b := Batch(s, 3)
+	require.Len(t, b, 3)
+	require.Len(t, b[0], 3)
+	require.Len(t, b[1], 3)
+	require.Len(t, b[2], 1)
+}
+
+type A int
+
+func (a A) String() string {
+	return strconv.Itoa(int(a))
+}
+
+func TestSortStrings(t *testing.T) {
+	assert.Equal(t, []A{1, 2, 5, 6}, SortStrings([]A{6, 2, 5, 1}))
```

Comment:
Just thinking out loud, not a change request cause it's doing exactly what it should be doing but numbers I don't think is a good idea to be sorted as a test if you put in a 10 (10, 2, 5, 1) it would output (1, 10, 2, 5) unless we had some string conversion handling as well?

Reply:
Also thinking out loud in reply:
I think that's a hypothetical that _if_ our test code input was different, then the test would fail.
But fixing it would mean changing `A`, and wouldn't change SortStrings implementation, and if it failed it'd be because someone changed the test.
So I reckon it's okay :)